mozilla/fx-private-relay
# fix #1967 MPP-1639: Add DomainAddress.used_on field

This PR fixes #1967.

How to test:
* Create a domain address
* Go to http://127.0.0.1:8000/api/v1/domainaddresses/
  * [ ] Verify you see a new `used_on` field for the domain address
